server/ws/testlib.go

```diff
@@ -0,0 +1,57 @@
+package ws
```

Comment:
This file is going to be compile as part of the regular package instead of the test package. Unless you need to export this, please change it to a file helpers_test.go, or something like that.

Reply:
Doesn't that also apply to everything in `services/store/storetests/`?  Also, why do we put store unit tests in a separate directory?